host-controller/src/main/java/org/jboss/as/host/controller/HostControllerEnvironment.java

```diff
@@ -234,6 +235,7 @@ public class HostControllerEnvironment extends ProcessEnvironment {
 
     private final RunningMode initialRunningMode;
     private final Stability stability;
+    private final Set<Stability> stabilities;
```

Comment:
This field shouldn't be needed as this.productConfig can be called to get the stabilities set.